operator/openshift/tests/operator-subscription.yaml

```diff
@@ -6,6 +6,6 @@ metadata:
 spec:
   channel: alpha
   name: seldon-operator
-  source: seldon-operators
+  source: seldon-core-catalog
   sourceNamespace: marketplace
-  startingCSV: seldon-operator.v1.1.0
+  startingCSV: seldon-operator.v1.2.2
```

Comment:
Is this expected to be 1.2.2 instead of 1.3?

Reply:
1.2.2 is last official release. Actually always good we leave this at 1.2.2 as we can check auto upgrades